src/Components/QuestionList/QuestionCard/QuestionCardS.scss

```diff
@@ -3,32 +3,36 @@
 .listBtn {
     margin: 1rem;
 }
+.QuestionCard {
+    .listItem.card {
+        display: flex;
+        flex-direction: column;
+        justify-content: flex-start;
+        width: 20rem;
 
-.listItem.card {
-    display: flex;
-    flex-direction: column;
-    justify-content: flex-start;
-    width: 20rem;
+        .card-body {
+            display: flex;
+            flex-direction: row;
+            justify-content: space-between;
+            align-content: center;
+        }
 
-    .card-body {
-        display: flex;
-        flex-direction: row;
-        justify-content: space-between;
-        align-content: center;
-    }
+        .difficulty {
+            overflow: hidden;
+            position: absolute;
+            top: 0;
+            left: 0;
+            width: 1rem;
+            height: 1rem;
+            // border-top: 4rem solid $warning;
+            border-right: 4rem solid transparent;
+            &.hard{
 
-    .difficulty {
-        overflow: hidden;
-        position: absolute;
-        top: 0;
-        left: 0;
-        width: 1rem;
-        height: 1rem;
-        // border-top: 4rem solid $warning;
-        border-right: 4rem solid transparent;
-    }
+            }
```

Comment:
this was from the morning and should have been taken out along with the comment on it.